pandas/core/generic.py

```diff
@@ -2834,9 +2834,9 @@ def to_latex(self, buf=None, columns=None, col_space=None, header=True,
         ...                    'mask': ['red', 'purple'],
         ...                    'weapon': ['sai', 'bo staff']})
         >>> df.to_latex(index=False) # doctest: +NORMALIZE_WHITESPACE
-        '\\begin{tabular}{lll}\n\\toprule\n      name &    mask &    weapon
-        \\\\\n\\midrule\n   Raphael &     red &       sai \\\\\n Donatello &
-         purple &  bo staff \\\\\n\\bottomrule\n\\end{tabular}\n'
+        '\\begin{tabular}{lll}\n\\toprule\n     name &   mask &   weapon
```

Comment:
These test changes are concerning. Is this reverting an API change between 0.23.4 and 0.24.0?

Reply:
i think this is the leading space that is wrongly added due to this bug